javav2/example_code/sqs-jms/src/main/java/com/example/sqs/jms/basic/JmsSqsSyncFifo.java

```diff
@@ -0,0 +1,102 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.example.sqs.jms.basic;
+
+import com.amazon.sqs.javamessaging.AmazonSQSMessagingClientWrapper;
+import com.amazon.sqs.javamessaging.ProviderConfiguration;
+import com.amazon.sqs.javamessaging.SQSConnection;
+import com.amazon.sqs.javamessaging.SQSConnectionFactory;
+import jakarta.jms.JMSException;
+import jakarta.jms.Message;
+import jakarta.jms.MessageConsumer;
+import jakarta.jms.MessageProducer;
+import jakarta.jms.Queue;
+import jakarta.jms.Session;
+import jakarta.jms.TextMessage;
+import org.slf4j.Logger;
+import software.amazon.awssdk.services.sqs.SqsClient;
+import software.amazon.awssdk.services.sqs.model.CreateQueueRequest;
+import software.amazon.awssdk.services.sqs.model.QueueAttributeName;
+
+import java.util.Map;
+import java.util.UUID;
+
+public class JmsSqsSyncFifo {
+    private static final Logger LOGGER = org.slf4j.LoggerFactory.getLogger(JmsSqsSyncFifo.class);
+    private static final String QUEUE_NAME = "MyQueue-" + UUID.randomUUID() + ".fifo";
+
+    public static void main(String[] args) {
+        // Create a new connection factory with all defaults (credentials and region) set automatically.
+        SQSConnectionFactory connectionFactory = new SQSConnectionFactory(
```

Comment:
Strange seeing new in Java code. Just a comment. Its clear this is not part of the AWS SDK for Java V2